### DISCUSSION TITLE
Final output

### DISCUSSION THREAD

**Discussion by i-m-amit** [sepal-contrib/sdg_15.3.1#19] [Ideas]
The final output of this indicator is basically a table with values in percentage of land degraded to total land area. 
We can use the `geemap.zonal_statistics_by_group` to get the table plot-wise. How can we implement this in SEPAL?

**Comment by 12rambau** [comment]
geemap is already installed in sepal. 

The main dificulty is too know in advance the column and name numbers. 
Best way is to test it in a notebook and write a dataframe (pandas) or geodataframe (geopandas) from the output dictionarie

I use it in this repository : https://github.com/openforis/zonal-analysis/blob/c597402be6d431a98378898a8369388bfe1cbdd9/scripts/zonal_computation.py#L114

**Comment by i-m-amit** [comment]
I will check it, however, `geemap.zonal_statistics_by_group` does not require the column and name numbers.

I think the following lines of code is enogh for this operation.
```python
out_dir = os.path.join(os.path.expanduser('~'), 'downloads')
indicator_15_3_1_stats = os.path.join(out_dir, aoi_name + '_indicator_15_3_1.csv')
zonal_statistics_by_group(
    in_value_raster = indicator_15_3_1,
    in_zone_vector = aoi_io.get_aoi_ee(), # it calculates the stats for all the features in a ee.FeatureCollection
    out_file_path = indicator_15_3_1_stats,
    statistics_type = "PERCENTAGE",
    decimal_places=2,
    denominator=1.0,
    scale=None,
    crs=None,
    tile_scale=1.0,
)
```
 What do you think?

**Reply by 12rambau** [reply]
- If the value is the defaulted `None`, it's not required to wirte it (`scale`, `crs`)
- for the naming use the more pythonic `f'{aoi_name}_indicator_15_3_1.csv'` ([documentation](https://www.datacamp.com/community/tutorials/f-string-formatting-in-python?utm_source=adwords_ppc&utm_campaignid=898687156&utm_adgroupid=48947256715&utm_device=c&utm_keyword=&utm_matchtype=b&utm_network=g&utm_adpostion=&utm_creative=229765585183&utm_targetid=dsa-429603003980&utm_loc_interest_ms=&utm_loc_physical_ms=1008736&gclid=Cj0KCQiA9P__BRC0ARIsAEZ6iriHCAwTZQ5JLDR_RxXL9Jvfbvka8-dsfUOfTwTJINjh695jPYw-HnUaAveBEALw_wcB))

The rest seems perfect
